servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/XdsHealthIndicator.java

```diff
@@ -261,9 +262,10 @@ public final void cancel() {
     @Override
     public String toString() {
         return "XdsHealthIndicator{" +
-                ", consecutive5xx=" + consecutive5xx.get() +
+                "consecutive5xx=" + consecutive5xx.get() +
                 ", successes=" + successes.get() +
```

Comment:
Consider moving `successes` to the top. Might be easier to read when all sorts of issues are next to each other.